rgd/settings.py

```diff
@@ -13,6 +13,26 @@
 from configurations import values
 
 
+class GdalMinioMixin(ConfigMixin):
+    """Setup S3-like environment variables for Minio setups."""
+
+    @staticmethod
+    def before_binding(configuration: Type[ComposedConfiguration]):
+        # `MINIO_STORAGE_ACCESS_KEY` and `MINIO_STORAGE_SECRET_KEY`
+        # can only be accessed via environment variables
+        os.environ.setdefault(
```

Comment:
It's kind of an abuse to use this function to do anything other than patch up dynamic settings. If you want to do something when the app starts up, I'd suggest this: https://docs.djangoproject.com/en/3.1/ref/applications/#django.apps.AppConfig.ready

Reply:
We only want this behavior when using [django-composed-configuration's `MinioStorageMixin`](https://github.com/girder/django-composed-configuration/blob/e65bdc513d74ead19035a837079dad6df5ddf48c/composed_configuration/_storage.py#L14). I am super uninformed on best practice regarding `ComposedConfiguration`. Not sure if `AppConfig.ready` seems right since this behavior shouldn't be run on production. Thoughts?